migration/v4/migrate.go

```diff
@@ -0,0 +1,26 @@
+package v4
+
+import (
+	liberr "github.com/konveyor/controller/pkg/error"
+	"github.com/konveyor/controller/pkg/logging"
+	"github.com/konveyor/tackle2-hub/migration/v4/model"
+	"gorm.io/gorm"
+)
+
+var log = logging.WithName("migration|v4")
+
+type Migration struct{}
+
+func (r Migration) Apply(db *gorm.DB) (err error) {
```

Comment:
This migration is only adding models and columns, so automigrate should be adequate.